addons/phone_validation/models/mail_thread_phone.py

```diff
@@ -121,3 +139,19 @@ def _phone_set_blacklisted(self):
 
     def _phone_reset_blacklisted(self):
         return self.env['phone.blacklist'].sudo()._remove([r.phone_sanitized for r in self])
+
+    def action_remove_phone_from_blacklist(self):
```

Comment:
Same remarks for phone than on mail, if you change something simply ensure we propagate here for symmetry. Like having pens parallel to the table edge.

Reply:
Done as suggested: `phone_action_blacklist_remove`